### ISSUE TITLE
Arguments spread over multiple lines

### ISSUE THREAD

**Issue by edgargressie** [AndyOGo/stylelint-declaration-strict-value#56]
Another issue I found is when using a function with the arguments spread over multiple lines. Not sure if this is related or I should create a new issue.

This, for example, will fail:

```
background-color: get-color(
  $arg1,
  $arg2
);
```

_Originally posted by @tvooo in https://github.com/AndyOGo/stylelint-declaration-strict-value/issues/11#issuecomment-386592198_

**Comment by AndyOGo** [comment]
Thank you.

This plugin should only detect the function usage, ignoring any other formatting rules like spacing, new lines, etc.

I suppose this fails right now.
So I will add tests here and create a PR.
https://github.com/AndyOGo/stylelint-declaration-strict-value/blob/master/test/default.js#L14-L28

**Comment by edgargressie** [comment]
Not sure if you noticed the issue reported by @tvooo, but I think it's still an issue. I used this a lot especially when nesting multiple (SCSS) functions and long variable names. But a combination of `calc` and `var` will fail too:

```css
margin: calc(
	var(--x) *
	var(--y)
)
```

**Comment by edgargressie** [comment]
Thanks! Feel free to contact me if you need help with testing it.

**Comment by AndyOGo** [comment]
@edgargressie 
Thank you very much to bring this to my attention again, I lost that [#11 comment ](https://github.com/AndyOGo/stylelint-declaration-strict-value/issues/11#issuecomment-386592198)

**Comment by edgargressie** [comment]
No problem, it works perfectly and saves a lot of ignore comments 👍  Thank you for your fast response and fix! 🚀

**Comment by AndyOGo** [comment]
@edgargressie 
My pleasure, you are welcome.

I just released a follow up patch [`v1.2.3`](https://www.npmjs.com/package/stylelint-declaration-strict-value/v/1.2.3) with support for multi line [CSS variables](https://developer.mozilla.org/en-US/docs/Web/CSS/var) :)

```css
.foo {
  color: var(
    --color-red,
    red
  );
}
```
